Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -108,7 +96,8 @@ int EmbedMolecule(ROMol &mol, unsigned int maxIterations = 0, int seed = -1,
                   bool ignoreSmoothingFailures = false,
                   bool enforceChirality = true,
                   bool useExpTorsionAnglePrefs = false,
-                  bool useBasicKnowledge = false, bool verbose = false,
+                  bool useBasicKnowledge = false,
+				  int ETversion = 1, bool verbose = false,
```

Comment:
Now that we have the versions that take the `EmbedParameters` object, I wonder if it's really worth continuing to add arguments to these functions. I'd argue that we should freeze the API on these and just support new options via `EmbedParameters`.